src/test/java/org/opensearch/flowframework/util/EncryptorUtilsTests.java

```diff
@@ -190,4 +190,13 @@ public void testEncryptDecryptTemplateCredential() {
         assertNotNull(decryptedCredential);
         assertEquals(testCredentialValue, decryptedCredential);
     }
+
+    public void testRedactTemplateCredential() {
+        // Redact template with credential field
+        Template processedTemplate = encryptorUtils.redactTemplateCredentials(testTemplate);
+
+        // Validate the credential field has been removed
+        WorkflowNode node = processedTemplate.workflows().get("provision").nodes().get(0);
+        assertNull(node.userInputs().get(CREDENTIAL_FIELD));
```

Comment:
(Style) would be nice to show that this key did exist before the redaction method, so assertNotNull at the top of this method.